### ISSUE TITLE
what does this do?

### ISSUE THREAD

**Issue by Doombotino** [sudoguy/tiktokpy#267]
I ran it and it gave some sort of info and it said it was liking videos i dont even know what this is doing.....

**Comment by warshell** [comment]
yeah please i want to know to, it seems liking and unlinking trending videos, the same with following, the nickname in quickstart is justinbiber? do i need to change this ? it getting trending videos from this feed? thanks for explanation.
